README.md

```diff
@@ -141,6 +142,7 @@ gnag {
 gnag {
     isEnabled = true
     setFailOnError(true)
+    useGitHubStatuses(true)
```

Comment:
I'm thinking this property should belong inside the `github` block as it's a setting specific to GitHub and not the entire `Gnag` plugin. I'm thinking to the future if we ever add support for other source repositories such as GitLab.